src/formatter/config/Config.hx

```diff
@@ -1,10 +1,10 @@
 package formatter.config;
 
+#if (sys || nodejs)
 import sys.FileSystem;
 import sys.io.File;
-#if !macro
```

Comment:
not sure if we can drop that, @Gama11 can you remember why we put it in conditionals?

Reply:
Looks like that was me in https://github.com/HaxeCheckstyle/haxe-formatter/commit/e3b345303a8a7a0bde6e26582dfbca3d9f60ca11... I guess it's ok to remove it as long as CI is green?

Reply:
I thought it might be something compilation cache or language server related, but if you are fine with deleting it, I won't object...